newIDE/app/src/stories/everything-else.stories.js

```diff
@@ -2523,8 +2523,8 @@ storiesOf('Profile/CreateProfile', module)
   .addDecorator(muiDecorator)
   .add('default', () => (
     <CreateProfile
```

Comment:
(#sorry) maybe an opportunity to move those 3 outside of the `everything-else.stories.js` ?